### PR TITLE
Create Dissolution Session interface

### PR DESCRIPTION
**Pull request by mateuszc-kainos**
Create tests
Update version for node-session-handler and web-security-node

### CODE REVIEW

File: src/models/dissolutionSession.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export interface DissolutionSession {
```
Comment (peadarkelly) [inline src/models/dissolutionSession.ts:1]: Hmmm, should we make this default as we don;t expect to need to export other things from this file

File: src/services/session/session.service.ts [hunk 2]
```diff
@@ -6,10 +6,19 @@ import { ISignInInfo } from 'ch-node-session-handler/lib/session/model/SessionIn
...
+import { DissolutionSession } from 'app/models/dissolutionSession'
+import Optional from 'app/models/optional'
+
 @provide(SessionService)
 export default class SessionService {
 
   public getAccessToken(req: Request): string {
     return req.session!.get<ISignInInfo>(SessionKey.SignInInfo)!.access_token!.access_token!
   }
+  public getDissolutionSession(req: Request): Optional<DissolutionSession> {
```
Comment (peadarkelly) [inline src/services/session/session.service.ts:18]: add spaces between methods you mad man

File: src/services/session/session.service.ts [hunk 3]
```diff
@@ -6,10 +6,19 @@ import { ISignInInfo } from 'ch-node-session-handler/lib/session/model/SessionIn
...
+    return req.session!.getExtraData<DissolutionSession>('dissolution')
```
Comment (peadarkelly) [inline src/services/session/session.service.ts:19]: I don't think you need a SessionKey enum, but just make this a constant at the top of the file:
```
private readonly DISSOLUTION_SESSION_KEY: string = 'dissolution'
```

File: test/services/session/session.service.test.ts [hunk 4]
```diff
@@ -35,4 +38,34 @@ describe('SessionService', () => {
...
+    it('should retrieve the dissolution object from the session', () => {
+      const dissolutionSession: DissolutionSession = generateDissolutionSession()
+      dissolutionSession.companyNumber = COMPANY_NUMBER
+
+      const req: Request = generateRequest()
+      req.session!.getExtraData = getSessionStub.withArgs('dissolution').returns(dissolutionSession)
+
+      const result: Optional<DissolutionSession> = service.getDissolutionSession(req)
+
+      assert.equal(result!.companyNumber, COMPANY_NUMBER)
```
Comment (peadarkelly) [inline test/services/session/session.service.test.ts:52]: I think you can just `assert.equal(result, dissolutionSession)`

File: src/services/session/session.service.ts (hunk 2 above)
Comment (mateuszc-kainos) [reply src/services/session/session.service.ts:18]: :D

File: src/models/dissolutionSession.ts (hunk 1 above)
Comment (faggc) [reply src/models/dissolutionSession.ts:1]: I agree, I can't see anything else being added to this

**Comment by faggc** [comment]
Looks good, just Peadar's comments to be actioned

**Pull request merged**